rts/lra/client/src/main/resources/META-INF/services/javax.ws.rs.client.ClientBuilder

```diff
@@ -1 +1,2 @@
-org.jboss.resteasy.microprofile.client.impl.MpClientBuilderImpl
+org.jboss.resteasy.client.jaxrs.ResteasyClientBuilder
+org.jboss.resteasy.microprofile.client.impl.MpClientBuilderImpl
```

Comment:
`MpClientBuilderImpl` was added because of Quarkus-based applications. We'll keep `MpClientBuilderImpl` among possible `ClientBuilder` implementations to allow Quarkus users to use lra-client out of the box.